builtin/game/chat.lua

```diff
@@ -1056,7 +1056,7 @@ core.register_chatcommand("last-login", {
 			param = name
 		end
 		local pauth = core.get_auth_handler().get_auth(param)
-		if pauth and pauth.last_login then
+		if pauth and pauth.last_login ~= -1 then
```

Comment:
Isn't it possible that last_login is still nil for old auth entries?

Reply:
How old are we talking about? As far as I understand, create_auth has always set the last login to os.time(), at least as far back as 0.4.14. Correct me if I'm wrong, but I don't think it's possible for it to be nil unless the database is corrupted.

https://github.com/sorcerykid/minetest/blob/24e8b0ac1ea45719937948607259f13866c8bc64/builtin/game/auth.lua#L129

Reply:
Right. This field was not created for accounts in old 0.4.x releases, thus still does not exist for players that never logged in since then. Needs checking what value `last_login` has (nil?).

Reply:
Considering the old code had `pauth.last_login` I'd just change this to `pauth.last_login and pauth.last_login ~= -1`

Reply:
Yes, good point @sfan5 and @SmallJoker, I was just thinking that myself. Doesn't hurt to check anyway. Will amend accordingly :)

Reply:
Bump.

Reply:
May apologies, I've just been super busy the past few days, but I'm planning on getting this completed today.